com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs

```diff
@@ -374,7 +410,13 @@ public override void Setup(ScriptableRenderContext context, ref RenderingData re
             }
 #endif
 
-            if (usesRenderPass)
+#if !UNITY_ANDROID && !UNITY_IOS
```

Comment:
What about desktop TBDR such as Apple Silicon and Windows ARM devices?

Reply:
I hadn't considered TBDR hardware on non-Android and non-iOS devices. I'm expanding this check and moving it to the constructor. Not sure if I have the vender ID's right however.